src/openloco/interop/interop.hpp

```diff
@@ -14,14 +14,14 @@
 #define FORCE_ALIGN_ARG_POINTER
 #endif
 
-#ifdef COMPAT_STD_BYTE
+//#ifdef COMPAT_STD_BYTE
```

Comment:
Should probably not redefine `std::byte`.

Reply:
Oh right, I meant to leave that as is. This should be probably wrapped with MSC_VER

Reply:
I think it's been introduced just for compatibility with macOS. Current versions of GCC and Clang also support `std::byte`.